### DISCUSSION TITLE
keyword2go mapping file update

### DISCUSSION THREAD

**Discussion by Antonialock** [geneontology/helpdesk#426] [General]
Hello,
We need the  keyword2go mapping file updated
http://current.geneontology.org/ontology/external2go/uniprotkb_kw2go

who can do this?
Thanks!

**Comment by kltm** [comment]
@Antonialock Currently, this mapping is directly maintained at an upstream at https://ftp.ebi.ac.uk/pub/databases/GO/goa/external2go/uniprotkb_kw2go (see https://github.com/geneontology/go-ontology/blob/1b477c66e31fad19d3f49beae423b05ad5eae7e5/src/ontology/Makefile#L560). @pgaudet , do you know who would be the best person to ping about this?

**Comment by Antonialock** [comment]
yes thanks, it can be resolved our end by Alex.

**Comment by Antonialock** [comment]
(this can close, I don't have permission)

**Reply by kltm** [reply]
This is a "discussion", not an "issue", so there is no closing.

**Reply by arnaudxk** [reply]
Hi, 
EBI file has been updated a few weeks ago:
   => https://ftp.ebi.ac.uk/pub/databases/GO/goa/external2go/uniprotkb_kw2go
I can not see the GO copy of this file updated yet
   => http://current.geneontology.org/ontology/external2go/uniprotkb_kw2go
   
Do you know when/how it is going to be updated ?
Thanks, 
Arnaud.

**Reply by kltm** [reply]
The last release was January 1st. We are currently processing a new release, which should hopefully be out this week.
